src/ngx_http_lua_module.c

```diff
@@ -661,6 +680,62 @@ ngx_http_lua_init(ngx_conf_t *cf)
         lmcf->postponed_to_access_phase_end = 0;
     }
 
+    len = lmcf->lua_cpath.len;
+
+    if (lmcf->requires_append_lua_cpath) {
+        for (i = 0; i < lmcf->append_lua_cpath->nelts; i++) {
+            p = (ngx_str_t *) lmcf->append_lua_cpath->elts;
+            len += p[i].len;
+        }
+
+        path = (u_char *) ngx_pcalloc(cf->pool, len);
```

Comment:
The `path` variable is usually named `p` in the world of nginx according to its usage.